src/backends/plotly.jl

```diff
@@ -409,6 +409,13 @@ plotly_surface_data(series::Series, a::AbstractVector) = a
 plotly_surface_data(series::Series, a::AbstractMatrix) = transpose_z(series, a, false)
 plotly_surface_data(series::Series, a::Surface) = plotly_surface_data(series, a.surf)
 
+#ensures that a gradient is called if a single color is supplied where a gradient is needed (e.g. if a series recipe defines marker_z)
+as_gradient(grad::ColorGradient, α) = grad
+as_gradient(grad, α) = cgrad(alpha = α)
+
+# allows passing a ColorGradient where a single color is expected - the other backends allow this
+PlotUtils.rgba_string(cg::ColorGradient) = rgba_string(cg[1])
```

Comment:
This may fit better in PlotUtils, but it feels a little like a hack that only plotlyjs uses